libraries/Ethernet/examples/udpntpclient_pedantic/RFC1305.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * see https://www.eecis.udel.edu/~mills/database/rfc/rfc1305/rfc1305c.pdf
+ * https://tools.ietf.org/html/rfc1305
+ */
+
```

Comment:
It is considered common good practice to start every header file with include guards.